src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      data-qa="big"
+      class="search-bar search-bar--big"
+    >
```

Comment:
The form tag is self-closed, which is not valid in HTML5. You should use a separate closing tag `</form>` instead of self-closing the form tag.

Reply:
I closed this tag on line 31